NitroxServer/Program.cs

```diff
@@ -31,11 +31,15 @@ static void Main(string[] args)
                 return;
             }
 
-            ConsoleCommandProcessor CmdProcessor = NitroxServiceLocator.LocateService<ConsoleCommandProcessor>();
-            while (server.IsRunning)
-            {
-                CmdProcessor.ProcessCommand(Console.ReadLine());
+            if (Environment.UserInteractive)

```

Comment:
Maybe we should add a comment about what this is fixing.

Reply:
Comment added, also added a comment to UdpServer.cs to avoid confusion that the `Listen` thread is running in the foreground.